### ISSUE TITLE
Functional testing doc - beta release

### ISSUE THREAD

**Issue by tselit** [interledger/web-monetization-extension#290]
Document the testing strategy for the WM extension's **beta** release.
Related to https://github.com/interledger/web-monetization-extension/issues/215

There are 2 different docs to guide testing, depending on the type of user:
1. **This ticket is for the Functional Testing Doc** (this is for ILF and WM community users who typically have more technical insight into the extension): https://docs.google.com/document/d/1CJOBlQjK1Zmdp1K6Pe2DP83uueTgJsBQSJbmDzYDrWw/edit
2. There is a separate ticket for usability testing (external users where the concern is UX and usability rather than tech functionality): https://github.com/interledger/web-monetization-extension/issues/239

**Comment by tselit** [comment]
Peer reviews of draft 1 started today, 30 Jul.

**Comment by tselit** [comment]
here's the markdown work-in-progress branch: [test/TestPlan.md](https://github.com/interledger/web-monetization-extension/blob/daeb026ec4518deef1e6f962b7831fd446c53d86/test/TestPlan.md)

**Comment by RabebOthmani** [comment]
@tselit this is done right? Can we close this ticket?

**Comment by tselit** [comment]
> @tselit this is done right? Can we close this ticket?

Thanks for checking @RabebOthmani . 
Content is complete. I'm busy moving to markdown, so the [working GitHub branch](https://github.com/interledger/web-monetization-extension/blob/daeb026ec4518deef1e6f962b7831fd446c53d86/test/TestPlan.md) is not yet merged.

**Comment by RabebOthmani** [comment]
What's the status on this please?

**Comment by tselit** [comment]
Hi @RabebOthmani [PR596](https://github.com/interledger/web-monetization-extension/pull/596) is still cooking. It has been cycling through review feedback & updates since the last update...there were many many comments 😄 
Almost there...should be in for further review 26 Sep.

**Comment by tselit** [comment]
The test plan is finally approved by 1 reviewer.
Awaiting approval from 1 more reviewer.

**Comment by tselit** [comment]
Received feedback from 2nd review.
Task was blocked by priority focus on the 2024 Summit, from 16 Oct to 1 Nov.

**Comment by tselit** [comment]
final round of review-updates in progress
